README.md

```diff
@@ -54,9 +53,9 @@ care of setting up a production ready Highly Available cluster!
 ### Native Support For The Most Popular Providers
 
 KubeOne natively supports the most popular providers, including AWS, Azure,
-DigitalOcean, GCP, Hetzner Cloud, Nutanix, OpenStack, and VMware vSphere. The
-natively supported providers enjoy additional features such as integration with
-Terraform and Kubermatic machine-controller.
+DigitalOcean, GCP, Hetzner Cloud, Nutanix, OpenStack, VMware vCloud Director,
```

Comment:
I think mentioning vCloud Director in the readme before providing actual integration is a bit too optimistic :)